configuration_manager/configuration_settings.xml

```diff
@@ -1,11 +1,27 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <settings>
-    <launcher_configurations>
-        <foo>
-            <bar></bar>
-        </foo>
-        <test_type>cluster</test_type>
-    </launcher_configurations>
+    <orchestrator>
+        <executors>
+          <local>
+            <command>mpirun</command>
+            <n_tasks_option>-n</n_tasks_option>
+            <n_tasks_default_value>2</n_tasks_default_value>
+            <get_version_option>-V</get_version_option>
+          </local>
+          <cluster>
+            <command>srun</command>
+            <n_tasks_option>-n</n_tasks_option>
+            <n_tasks_default_value>2</n_tasks_default_value>
+            <get_version_option>--version</get_version_option>
+          </cluster>
+        </executors>
+        <simulation_models>
+          <models_dir>${CO_SIM_INSTALL_DIR}/models/</models_dir>
+        </simulation_models>
+        <broker_translators>
+          <translators_dir>${CO_SIM_INSTALL_DIR}/translators/</translators_dir>
+        </broker_translators>
```

Comment:
This should be 'transformers' now, right!?

Reply:
I agree!!! I will change it out, and later merge it.